Source/JavascriptEditor/JavascriptEditorViewport.cpp

```diff
@@ -70,6 +124,19 @@ class FJavascriptEditorViewportClient : public FEditorViewportClient
 		}
 	}
 
+	virtual void Draw(FViewport* Viewport, FCanvas* Canvas) override
+	{
+		FEditorViewportClient::Draw(Viewport, Canvas);
+
+		if (Widget->SelectionBeginPoint.X >= 0)
+		{
+			FCanvasBoxItem Box(Widget->SelectionBeginPoint, Widget->SelectionEndPoint - Widget->SelectionBeginPoint);
+			Box.SetColor(FColor::White);
+			Box.LineThickness = 1.0;
+			Box.Draw(Canvas);
```

Comment:
Yes, I fully agree, I updated my pull request.

I exposed the `FCanvas` as a `UCanvas` like in `JavascriptEdMode`.

For this purpose, I moved `GetCanvasByName()` from `JavascriptEdMode.cpp` to `JavascriptEditorLibrary.h` to be able to access it from `JavascriptEdMode.cpp` and `JavascriptEditorViewport.cpp`.

Tell me if there is a better solution.